### PR TITLE
[FE] [ADD] Eip 237 FE cookie banner

### PR DESCRIPTION
**Pull request by wernerramon**
### 🛠 Changes being made

This will add the cookie banner, cookie statement and list of technologies to resto web and visitor web

### ✨ What's the context?

What's the context for the changes? Are there any

### 🧠 Rationale behind the change

Why did you choose to make these changes? Were there any trade-offs you had to consider?

### 🧪 Test plan

cypress tests

### 📸 Screenshots (optional)

If you made UI changes, what are the before an afters?

### 🏎 PR checklist

1. Read the ticket
- [ ] Read the related ticket on Jira and understand the requirements for this ticket. In case anything is unclear, ask the ticket creator or ticket executor.
2. Matching PR description
- [ ] Check that the PR description on GitHub is aligned with the ticket description on Jira.
3. Review the changes on GitHub
- [ ] is the current version of the destination branch (mostly main) merged into the PR branch?
- [ ] does it fulfill the definition of done for the ticket?
- [ ] were the tests adjusted or extended for the changed behavior?
- [ ] was the documentation adjusted or extended for changed behavior?
- [ ] does it align with the team code guidelines?
- [ ] check readability of code - is it easy to understand what is happening in the code snippets?
- [ ] proper naming
- [ ] is it logical?
- [ ] are there any performance issues?
- [ ] could it be improved / simplified?
4. Execute locally / in testing environment
- [ ] Execute the code locally and check if there are any issues. Test whether the new behavior from the ticket acts as expected. Try to think of edge cases and test them.
5. Run tests
- [ ] Run the tests locally and check that all of them succeed.
6. Linting & Formatting
- [ ] Check that the changes didn’t cause any linting or formatting warnings of errors.
7. Submit Feedback
- [ ] Approve if all of the following points fulfilled:
   - all previous steps did not reveal any issues
   - there are no open comments on the PR

- [ ] otherwise, request changes with specific comments (constructive) on what to change!

### CODE REVIEW

File: packages/restoWeb/src/pages/CookieStatement/CookiePage.tsx [hunk 1]
```diff
@@ -0,0 +1,242 @@
...
+
+
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.content}>
+        <h2 className={styles.title}>{t('pages.CookieStatement.title')}</h2>
+        <hr className={styles.line} />
+        <div className={styles.text}>
+          <p>{t('pages.CookieStatement.intro')} <a href='/privacystatement'>{t('pages.CookieStatement.privacy')}</a>.</p>
```
Comment (JosefineM2000) [inline packages/restoWeb/src/pages/CookieStatement/CookiePage.tsx:130]: correct link is just /privacy

File: packages/visitorWeb/src/pages/CookieStatement/CookiePage.tsx [hunk 2]
```diff
@@ -0,0 +1,242 @@
...
+
+  return (
+    <div className={styles.container}>
+      <div className={styles.content}>
+        <h2 className={styles.title}>{t('pages.CookieStatement.title')}</h2>
+        <hr className={styles.line} />
+        <div className={styles.text}>
+          <p>{t('pages.CookieStatement.intro')} <a href='/privacy'>{t('pages.CookieStatement.privacy')}</a>.</p>
+          <br />
+          <p>{t('pages.CookieStatement.intro2')}</p>
```
Comment (JosefineM2000) [inline packages/visitorWeb/src/pages/CookieStatement/CookiePage.tsx:132]: in this text, the our email is mentioned. I think it would we nice to link it with the contact us page.

File: packages/visitorWeb/src/pages/CookieStatement/CookiePage.tsx [hunk 3]
```diff
@@ -0,0 +1,242 @@
...
+          <p className={styles.italic}>{t('pages.CookieStatement.Marketing')}</p>
+          <br />
+          <p>{t('pages.CookieStatement.marketing')}</p>
+        </div>
+        <h2 className={styles.title}>{t('pages.CookieStatement.Security')}</h2>
+        <hr className={styles.line} />
+        <div className={styles.text}>
+          <p className={styles.italic}>{t('pages.CookieStatement.security')}</p>
+          <br />
+          <p>{t('pages.CookieStatement.securityTxt')}</p>
```
Comment (JosefineM2000) [inline packages/visitorWeb/src/pages/CookieStatement/CookiePage.tsx:193]: in this text, the our email is mentioned. I think it would we nice to link it with the contact us page.

File: packages/visitorWeb/src/pages/CookieStatement/CookiePage.tsx [hunk 4]
```diff
@@ -0,0 +1,242 @@
...
+          <p>{t('pages.CookieStatement.tips')}<a href='/privacy'>{t('pages.CookieStatement.privacy')}</a>{t('pages.CookieStatement.tips2')}</p>
+          </div>
+        <h2 className={styles.title}>{t('pages.CookieStatement.Conclusion')}</h2>
+        <hr className={styles.line} />
+        <div className={styles.text}>
+          <p>{t('pages.CookieStatement.conclusion1')}</p>
+          <br />
+          <p>{t('pages.CookieStatement.conclusion2')}</p>
+          <br />
+          <p>{t('pages.CookieStatement.conclusion3')}</p>
```
Comment (JosefineM2000) [inline packages/visitorWeb/src/pages/CookieStatement/CookiePage.tsx:230]: in this text, the our email is mentioned. I think it would we nice to link it with the contact us page.

**Review by JosefineM2000 — CHANGES_REQUESTED** [review]
Please apply the comments on restoWeb to the according one in visitorWeb as well.

File: packages/restoWeb/src/components/dumpComponents/popup/Popup.tsx [hunk 5]
```diff
@@ -44,8 +44,8 @@ export const Popup: React.FC<PopupProps> = (
     <PopupContainer>
       <div className={styles.PopupText} >{message}</div>
       <div>
-        <PopupButton onClick={onConfirm}>{t('common.confirm')}</PopupButton>
-        <PopupButton onClick={onCancel}>{t('common.cancel')}</PopupButton>
+        {/* <PopupButton onClick={onConfirm}>{t('common.confirm')}</PopupButton>
```
Comment (JosefineM2000) [inline packages/restoWeb/src/components/dumpComponents/popup/Popup.tsx:47]: I dont think this should be commented out

**Review by JosefineM2000 — CHANGES_REQUESTED** [review]

**Review by JosefineM2000 — APPROVED** [review]
